.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,72 @@
+name: Bug Report
+description: Create a report to help us improve
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        > _Thanks for filing a bug ticket. We appreciate your time and effort! If you need real-time support, consider joining us on Discord!_
+  - type: input
+    id: byond
+    attributes:
+      label: BYOND Version
+      description: What version of BYOND are you using?
+      placeholder: "514.1589"
+    validations:
+      required: false
+  - type: input
+    id: round-date
+    attributes:
+      label: Round Date
+      placeholder: "YYYY/MM/DD"
+    validations:
+      required: false
+  - type: input
+    id: round-id
+    attributes:
+      label: Round ID
+      placeholder: "12345"
+    validations:
+      required: false
+  - type: textarea
+    id: testmerges
+    attributes:
+      label: Test merges
+      description: A list of currently active test merges (you can get this from the Message of the Day when you connect to the server)
```

This should reference Show Server Revision

---

Most players will have no idea what that is. It's far more reliable to reference them towards the message of the day. That being said, it can be added as a secondary reference.

```suggestion
      description: A list of currently active test merges (you can get this from the Message of the Day when you connect to the server, or the `Show Server Revision` OOC verb)
```